Sources/Lumina/Camera/Extensions/Delegates/DepthDataExtension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DepthDataExtension.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+extension LuminaCamera: AVCaptureDepthDataOutputDelegate {
+  func depthDataOutput(_ output: AVCaptureDepthDataOutput, didOutput depthData: AVDepthData, timestamp: CMTime, connection: AVCaptureConnection) {
+    DispatchQueue.main.async {
+      self.delegate?.depthDataCaptured(camera: self, depthData: depthData)
+    }
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/Delegates/DepthDataExtension.swift#L18"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>DepthDataExtension.swift:18</code></td>
    </tr>
  </tbody>
</table>